source/manifest.json

```diff
@@ -34,6 +34,7 @@
 	"background": {
 		"scripts": [
 			"browser-polyfill.min.js",
+			"content.js",
```

Comment:
This is causing the background script to fail because some features have code outside `init`. I'll have to fix it